user-org-actor/src/main/java/org/sunbird/user/UserReadActor.java

```diff
@@ -48,7 +50,14 @@ public void onReceive(Request request) throws Throwable {
      */
     public void readUserById(Request request) throws BaseException {
         startTrace("readUserById");
-        Response response = userESDao.getUserById((String) request.getRequest().get(JsonKey.USER_ID));
+        Response response = null;
+        try {
+            response = userESDao.getUserById((String) request.getRequest().get(JsonKey.USER_ID));
+        } catch (Exception e) {
+            ProjectLogger.log("Exception occurred while reading user ES.", e);
+            userService = new UserServiceImpl();
```

Comment:
Use service factory , no need to create instance every places,

Reply:
Created factory for service object creation.